crates/ruma-common/src/events/relation.rs

```diff
@@ -154,8 +151,7 @@ pub struct Relations {
     pub replace: Option<BundledReplacement>,
 
     /// Thread relation.
-    #[cfg(feature = "unstable-msc3440")]
-    #[serde(rename = "io.element.thread", alias = "m.thread")]
+    #[serde(rename = "m.thread", alias = "io.element.thread")]
```

Comment:
Is `io.element.thread` used in practice? If not, maybe we should remove it because of #1266.

Reply:
Ah right, it should not be used anymore since a few versions of synapse now so we can probably remove it